docs/data/material/getting-started/templates/dashboard/Dashboard.tsx

```diff
@@ -32,30 +32,30 @@ export default function Dashboard() {
       <Box sx={{ display: 'flex' }}>
         <Navbar mode={mode} toggleColorMode={toggleColorMode} />
         <SideMenu />
+        {/* Main content */}
         <Box
           component="main"
           sx={(theme) => ({
-            backgroundColor: alpha(theme.palette.background.paper, 0.4),
             flexGrow: 1,
-            height: '100vh',
+            top: '64px',
```

Comment:
I'd remove this. The white space works fine without the `top:'64px'`

Current:

<img width="969" alt="Screenshot 2024-06-10 at 11 21 13" src="https://github.com/mui/material-ui/assets/37222944/f61aec54-37e5-4405-88fc-c36c3dff75d0">
<img width="973" alt="Screenshot 2024-06-10 at 11 21 42" src="https://github.com/mui/material-ui/assets/37222944/938f0009-89cf-4086-bf0f-a9cd0049fab2">


Removing `top: '64px'` :

<img width="977" alt="Screenshot 2024-06-10 at 11 25 06" src="https://github.com/mui/material-ui/assets/37222944/90d54003-c9a3-4d4f-96b7-c4bc5c6f2aa3">

Reply:
I removed the top padding instead, which was giving us the extra spacing. the `top:64px` is there to help us fix the scrollbar origin :)